.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+node_js:
+- '8'
+matrix:
+  include:
+  - os: linux
+    dist: xenial
+    sudo: required
+  - os: osx
+cache:
+  yarn: true
+notifications:
+  email: false
+script:
+- npm run build
+- npm run build:electron
+deploy:
+  provider: releases
+  api_key:
+    secure: H72TbvbYGaf8vjRZvTZuXQMDxiOIuYskVQJQrGBF6B7F748SwH6UIBBkscXD1rVE1c55WhmuWIGGUZi1++wlhBHylwmBwfqwcbVykMiRMHMU6l9PEMIvxMOCyVZlBxxo0g3ULOJnvoakGC6NCxrJAl4jLIvRzZHMGACzTNpTei8=
```

Comment:
Are you sure you want to put this on GitHub?

Reply:
I was surprised but that's how they say we should do it... And env vars from the project settings in Travis are not set when a build is triggered by GitHub so there is no other choice

Also this is an encrypted key which can only be used by Travis, and it was generated by Travis CLI (a little modified by hand)